libs/ui/src/big_metric.tsx

```diff
@@ -0,0 +1,57 @@
+/* stylelint-disable order/properties-order */
+import React from 'react';
+import styled from 'styled-components';
+
+import { format } from '@votingworks/utils';
+
+import { H1 } from './typography';
+import { LabelledText } from './labelled_text';
+
+/** Props for {@link BigMetric}. */
+export interface BigMetricProps {
+  label: React.ReactNode;
+  value: number;
+
+  /** For backward compatibility with tests in VxScan */
+  valueElementTestId?: string;
+}
+
+const StyledContainer = styled.span`
+  display: inline-block;
+  font-size: 1rem;
+  line-height: 1;
+  margin-bottom: 0.35rem;
+  text-align: center;
+`;
+
+const StyledValue = styled(H1)`
+  display: inline-block;
+
+  /* TODO: Remove once VxScan is updated to default to the new VVSG size theme: */
+  font-size: ${(p) => (p.theme.sizeMode === 'legacy' ? '72px' : undefined)};
+  line-height: 1;
+  margin: 0;
+`;
+
+/**
+ * Displays a formatted, H1-sized numerical metric value with the given label.
+ */
+export function BigMetric(props: BigMetricProps): JSX.Element {
+  const { label, value, valueElementTestId } = props;
+
+  const formattedValue = format.count(value);
+  const formattedLabel = `${label}: ${formattedValue}`;
+
+  return (
+    <StyledContainer>
+      <LabelledText aria-hidden label={label}>
```

Comment:
Why do we want this to be `aria-hidden`? Based on the `aria-label` on the child element, it seems like we want this tree to be rendered on accessibility devices.

Reply:
Oh, good catch! I meant to put that on the label, not the whole component - will fix that up